cdn/dev/keyboard-search/install.js

```diff
@@ -61,6 +61,46 @@ function startAfterPageLoad_Windows(data) {
           location.href = downloadUrl;
       }, 1000);
 
+      window.addEventListener('blur', function() {
+        window.clearTimeout(fallbackHandle);
+       });
+    }
+  }, 10);
+}
+
+/**
+ * Redirects to keyman:// to trigger Keyman Configuration, and if that fails,
+ * to the standard download url for the package to install. This currently only
+ * applies to Linux users.
+ * @param {object} data from the object with properties for host, tier, version, id, bcp47, name
+ */
+function startAfterPageLoad_Linux(data) {
+  window.setTimeout(function() {
+    const platform = document.documentElement.getAttribute('data-platform'), browser = document.documentElement.getAttribute('data-browser');
+    if(platform == 'linux') {
+      const downloadKeymanUrl = '/linux/download';
+      const installKeyboardUrl =
+        "/keyboards/install/" + encodeURIComponent(data.id) +
+        (data.bcp47 == "" ? "" : "?bcp47=" + encodeURIComponent(data.bcp47));
+
+      const keymanUrl = buildStandardKeymanProtocolDownloadLink(
+        data.id, data.bcp47
+      );
+
+      location.href = keymanUrl;
+
+      const fallbackHandle = window.setTimeout(function() {
+        document.getElementById("content").innerHTML =
```

Comment:
I'd prefer to handle this with CSS controls so the HTML content is in the .PHP file, and we hide and show with CSS

Reply:
done